nachomemes/bot.py

```diff
@@ -107,124 +192,99 @@ async def on_message_edit(before: Message, after: Message):
     try:
         await ctx.trigger_typing()
-        is_hard = refresh_type == "--hard"
         config: GuildConfig = STORE.guild_config(ctx.guild)
         if is_hard:
             if not config.can_admin(_get_member(ctx)):
-                return await ctx.send(
-                    f"```{config.no_admin(_get_member(ctx))}```"
-                )
+                return await ctx.send(embed=Embed(
+                    title = title, 
+                    description = config.no_admin(_get_member(ctx))
+                ))
         else:
             if not config.can_edit(_get_member(ctx)):
-                return await ctx.send(
-                    f"```{config.no_admin(_get_member(ctx),'refresh','edit')}```"
-                )
-
-        message = STORE.refresh_memes(config.guild_id, is_hard)
-        await ctx.send(f"```{message}```")
+                return await ctx.send(embed=Embed(
+                    title = title, 
+                    description = config.no_admin(_get_member(ctx),'refresh','edit')
+                ))
+        return await ctx.send(embed=Embed(
+                title = title, 
+                description = STORE.refresh_memes(config.guild_id, is_hard)
+        ))
     except Exception as ex:
         await report(ctx, ex)
 
+
 @memebot.command(description="set the discord role for adminstrators")
 async def admin_role(ctx: Context, role_id: str=None):
     try:
         config: GuildConfig = STORE.guild_config(ctx.guild)
         if not role_id:
             role: Optional[Role] = ctx.guild.get_role(config.admin_role) if config.admin_role else None
-            await ctx.send(textwrap.dedent(
-                f"```Members of '{role}' are authorized to administer the memes.```"))
+            return await ctx.send(embed=Embed(
+                title = "Getting current discord admin role",
+                description = f"Members of '{role}' are authorized to administer the memes."
+            ))
         else:
             role = ctx.guild.get_role(int(role_id)) if role_id else None
-            message = config.set_admin_role(_get_member(ctx), role)
             STORE.save_guild_config(config)
-            await ctx.send(textwrap.dedent(f"```{message}```"))
+            return await ctx.send(embed=Embed(
+                title = "Setting discord admin role to " + role_id,
+                description = config.set_admin_role(_get_member(ctx), role)
+            ))
     except Exception as ex:
         await report(ctx, ex)
 
+
 @memebot.command(description="set the discord role for editors")
 async def edit_role(ctx: Context, role_id: str=None):
     try:
         config: GuildConfig = STORE.guild_config(ctx.guild)
         if not role_id:
             role: Optional[Role] = ctx.guild.get_role(config.edit_role) if config.edit_role else None
-            await ctx.send(textwrap.dedent(f"```Members of '{role}' are authorized to edit the memes.```"))
+            return await ctx.send(embed=Embed(
```

Comment:
We can make this a function I think with the one from line 243 and just take the action that we are talking about (`edit` here).